src/js/player.js

```diff
@@ -1640,6 +1640,40 @@ class Player extends Component {
   }
 
   /**
+   * Check whether the player can play a given mimetype
+   *
+   * @param {String} type The mimetype to check
+   * @return {String} 'probably', 'maybe', or '' (empty string)
+   * @method canPlayType
+   */
+  canPlayType(type) {
+    let can;
+
+    // Loop through each playback technology in the options order
+    for (var i=0,j=this.options_.techOrder;i<j.length;i++) {
```

Comment:
You should use a `let` here. Also, can you add some whitespace around the operators and after commas and semicolons?